### PR TITLE
Fix expired session || modify details page

### PR DESCRIPTION
**Pull request by Nicolaswg**
# Hi there 👋🏻

Thanks for taking the time to review the project. Here are some things important to note about it:

### Changes implemented in this PR

- [X] Fix session expired
- [X] Redirect user to `404` when wrong equipment Id is send it 

## Final notes 📔 

Is expected that this project meets the requirements. Please let me know if anything is left to be done or can be improved. Again, thanks for the review.

### CODE REVIEW

**Review by JohnFTitor — APPROVED** [review]
# Hi @Nicolaswg,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:



## Highlights


## Optional suggestions 


_Every comment with the [OPTIONAL] prefix is just a personal suggestion that I think can make your code better. I'm open to discussing your opinions around them but are not required so we can continue the development of the project. 
Cheers and Happy coding!:clap::clap::clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
This is a feature teammate review. Remember that everything said here is intended for both of us to learn and grow as Software Developers 💻  and we may have different approaches or perspectives. That being said, don't hesitate to comment if you disagree with one required change. We can talk everything through 🐱

File: src/logic/apiRequests.js [hunk 1]
```diff
@@ -28,6 +25,10 @@ export const deleteReservation = (id, equipmentId, date) => async (dispatch) =>
     return res.data;
   })
     .catch((err) => {
+      if (err.response?.status === 500) {
```
Comment (jose-Abel) [inline src/logic/apiRequests.js:28]: 500 status is for Internal Server Error, let's make it 401 for Unauthorized errors please.

**Review by jose-Abel — CHANGES_REQUESTED** [review]
# Hi @Nicolaswg
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there## Highlights## Required Changes :recycle:
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is just a personal suggestion that I think can make your code better. I'm open to discussing your opinions around them but are not required so we can continue the development of the project.

Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
------
This is a feature teammate review. Remember that everything said here is intended for both of us to learn and grow as Software Developers :computer:  and we may have different approaches or perspectives. That being said, don't hesitate to comment if you disagree with one required change. We can talk everything through.

**Review by jose-Abel — APPROVED** [review]
# Hi @Nicolaswg ,
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada:## Highlights## Optional suggestions_Every comment with the [OPTIONAL] prefix is just a personal suggestion that I think can make your code better. I'm open to discussing your opinions around them but are not required so we can continue the development of the project.
Cheers and Happy coding!:clap::clap::clap:Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.------

This is a feature teammate review. Remember that everything said here is intended for both of us to learn and grow as Software Developers :computer:  and we may have different approaches or perspectives. That being said, don't hesitate to comment if you disagree with one required change. We can talk everything through

**Pull request merged**